ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-master.yaml

```diff
@@ -27,6 +27,10 @@ tag_specification:
 tests:
 - as: kubevirt-must-gather-e2e-aws
   commands: |
+    git clone https://github.com/kubevirt/hyperconverged-cluster-operator
+    cd hyperconverged-cluster-operator
+    sed -i "s#15m#30m#g" ./deploy/kustomize/deploy_kustomize.sh
```

Comment:
I'm just wondering if we should make that the default value in the script

Reply:
I think we should, @orenc1 wdyt ?